scp/lib/HomePage.dart

```diff
@@ -67,38 +67,39 @@ class _HomePageState extends State<HomePage> {
               mainAxisAlignment: MainAxisAlignment.start,
               children: <Widget>[
                 DrawerHeader(
+                    margin: EdgeInsets.all(0),
```

Comment:
Umm, what is this line exactly doing?

Reply:
This line overrides the default margin of 8. I did this since the bottom part looked a bit clumsy and there was a lot of space above.